### DISCUSSION TITLE
Elevation of Chart

### DISCUSSION THREAD

**Discussion by passerby29** [patrykandpatrick/vico#446] [Questions]
I want to add a chart to another view, such as a button. But it is always on top even if I add button height = 100dp and chart height = 0dp.
![image](https://github.com/patrykandpatrick/vico/assets/78204469/77345b7d-b4af-4298-8a6f-b0627e33b559)

**Comment by Gowsky** [comment]
Hello @passerby29, the z-order of views is usually managed by their parent layout. We don't apply any custom elevation or z-translation. Without any more details, we cannot give you any guidance or help.
